stdlib/public/Concurrency/Task.swift

```diff
@@ -618,26 +639,26 @@ public struct UnsafeCurrentTask {
     self._task = task
   }
 
-  /// Returns `Task` representing the same asynchronous context as this 'UnsafeCurrentTask'.
-  ///
-  /// Operations on `Task` (unlike `UnsafeCurrentTask`) are safe to be called
-  /// from any other task (or thread).
+  /// The current task,
+  /// represented in a way that's safe to store for later use.
   public var task: Task {
     Task(_task)
   }
 
-  /// Returns `true` if the task is cancelled, and should stop executing.
+  /// A Boolean value that indicates whether the current task was canceled.
   ///
   /// - SeeAlso: `checkCancellation()`
   public var isCancelled: Bool {
     _taskIsCancelled(_task)
   }
 
-  /// Returns the `current` task's priority.
+  /// The current task's priority.
   ///
-  /// If no current `Task` is available, returns `Priority.default`.
+  /// If there is no current task,
+  /// the value of this property is `Priority.default`.
+  /// ◊TR: When would this happen,
+  /// ◊TR: rather than withUnsafeCurrentTask() passing nil as its closure's argument?
```

Comment:
Thanks for spotting this -- this will always return a value; the `static var currentPriority` needs the "if there is no current task" but the instance one does not.